README.md

```diff
@@ -83,9 +108,14 @@ The scope required for sending metrics is the `Ingest metrics` scope in the **AP
 
 The `Prefix` property specifies an optional prefix, which is prepended to each metric key, separated by a dot (`<prefix>.<namespace>.<name>`).
 
-#### Default Labels/Dimensions
+#### Default Dimensions
+
+The `DefaultDimensions` property can be used to optionally specify a list of key/value pairs, which will be added as additional dimensions to all data points.
+
+#### OneAgent metadata enrichment
 
-The `Tags` property can be used to optionally specify a list of key/value pairs, which will be added as additional labels/dimensions to all data points.
+If the `OneAgentMetadataEnrichment` property is set to true, the exporter attempts to [read metadata from a file](https://www.dynatrace.com/support/help/how-to-use-dynatrace/metrics/metric-ingestion/ingestion-methods/enrich-metrics/).
+Metadata read that way will be added as dimensions to all exported metrics.
 
 ### Known issues and limitations
```

Comment:
Are these still applicable?

Reply:
Seems to still be true, I did not find a way to distinguish between the two. There are only sums, I think this is something that has to be changed in the SDK.